### PR TITLE
Add documentation for rasa webchat

### PR DESCRIPTION
**Pull request by gausie**
**Proposed changes**:
- Add documentation for the Rasa Webchat Widget. This is pending the successful deployment of the storybook docs

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by gausie** [comment]
At last I got the full documentation working! So this can finally go in

File: docs/docs/connectors/your-own-website.mdx [hunk 1]
```diff
@@ -10,7 +10,7 @@ is usually the chat interface that ships with Rasa X, where you can [invite user
 to test your bot](https://rasa.com/docs/rasa-x/user-guide/share-assistant/#share-your-bot).
 
 If you already have an existing website and want to add a Rasa assistant to it,
-you can use [Chatroom](https://github.com/scalableminds/chatroom), a widget which you can incorporate into your existing webpage by adding a HTML snippet.
+you can use [Rasa Webchat](#webchat-widget), a widget which you can incorporate into your existing webpage by adding an HTML snippet.
```
Comment (TyDunn) [inline docs/docs/connectors/your-own-website.mdx:13]:
```suggestion
you can use [Rasa Webchat](#webchat-widget), a widget which you can incorporate into your existing webpage by adding an HTML snippet.
```
Did we confirm with marketing on the name `Rasa Webchat`?

File: docs/docs/connectors/your-own-website.mdx [hunk 2]
```diff
@@ -155,3 +155,18 @@ as a JSON object under the key `token`:
...
+
+Once you've set up your SocketIO channel you can use the official Rasa Webchat Widget on any webpage.
+Just paste the following into your site HTML and paste the URL of your Rasa instance into
+the `data-websocket-url` attribute
+
+```html
+<div id="rasa-webchat-widget" data-websocket-url="https://your-rasa-url-here/"></div>
+<script src="https://unpkg.com/@rasahq/rasa-webchat" type="application/javascript"></script>
+```
+
```
Comment (TyDunn) [inline docs/docs/connectors/your-own-website.mdx:168]: Can we add something about what / how you can customize certain attributes?

**Review by TyDunn — CHANGES_REQUESTED** [review]

File: docs/docs/connectors/your-own-website.mdx (hunk 1 above)
Comment (tmbo) [reply docs/docs/connectors/your-own-website.mdx:13]: didn't kick of the project so not sure. let me check in with them - good point.

File: docs/docs/connectors/your-own-website.mdx (hunk 2 above)
Comment (gausie) [reply docs/docs/connectors/your-own-website.mdx:168]: My intention was that that is covered in the next line.

Comment (TyDunn) [reply docs/docs/connectors/your-own-website.mdx:168]: Okay, can we make that more clear? Perhaps say that it is possible and then how you would do it for one attribute as an example?

File: docs/docs/connectors/your-own-website.mdx [hunk 3]
```diff
@@ -155,3 +155,18 @@ as a JSON object under the key `token`:
...
+Once you've set up your SocketIO channel you can use the official Rasa Chat Widget on any webpage.
+Just paste the following into your site HTML and paste the URL of your Rasa instance into
+the `data-websocket-url` attribute
+
+```html
+<div id="rasa-chat-widget" data-websocket-url="https://your-rasa-url-here/"></div>
+<script src="https://unpkg.com/@rasahq/rasa-chat" type="application/javascript"></script>
+```
+
+For more information, including how to fully customize the widget to fit in with your website, you can check out the [full documentation](https://chat-widget-docs.rasa.com/).
```
Comment (TyDunn) [inline docs/docs/connectors/your-own-website.mdx:169]:
```suggestion
For more information, including how to fully customize the widget for your website, you can check out the [full documentation](https://chat-widget-docs.rasa.com/).
```

File: docs/docs/connectors/your-own-website.mdx [hunk 4]
```diff
@@ -155,3 +155,18 @@ as a JSON object under the key `token`:
 }
 ```
 
+### Chat Widget
+
+Once you've set up your SocketIO channel you can use the official Rasa Chat Widget on any webpage.
```
Comment (TyDunn) [inline docs/docs/connectors/your-own-website.mdx:160]:
```suggestion
Once you've set up your SocketIO channel, you can use the official Rasa Chat Widget on any webpage.
```

**Review by TyDunn — APPROVED** [review]
Great work, Sam! LGTM

**Pull request merged**